kubernetes/aks/system/vouch-proxy/base/ingress.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: vouch-proxy
+  namespace: vouch-proxy
+  labels:
+    app.kubernetes.io/name: vouch
+    app.kubernetes.io/instance: vouch
+  annotations:
+    external-dns.alpha.kubernetes.io/target: inspection.alpha.canada.ca
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    kubernetes.io/tls-acme: "true"
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/proxy-buffer-size: 64k
```

Comment:
why 64k? most common value seems to be 8K

discussion: 

https://stackoverflow.com/questions/686217/maximum-on-http-header-values

Reply:
This is the size needed for authenticating with Azure. Anything less would result in limit errors when trying to login.

Reply:
ok, but where is the rationale for specific constant values being documented in the product?

to find the specific rationale here, a future maintainer would have to dig and find this conversation history. should this not be part of some documentation adjacent to the configuration file (or yaml comments)?